### PR TITLE
chore(revamp): remove unused code (#2670)

### PR DESCRIPTION
**Pull request by hervyt**
# Context
> Clean some unused code

## How to test
You should see no regression

## Related issues

- Related to #2670 

# Checklist

## Quality

- [X] I consider this work complete and ready for review
- [X] I tested all features and edge cases
- [ ] I refactored code where necessary to improve quality
- [ ] I made sure my code meets development guidelines
- [X] I performed a self-review of my code
- [ ] I added the `skip-feature-env` label if a feature environment is not needed for this PR

## Testing

- [ ] I wrote test cases for the relevant use cases (unit, integration or e2e tests)
- [ ] (Bug fixes only) I added tests that reproduce and verify the fix

### What tests have been made

- [ ] Unit tests
- [ ] Integration tests
- [ ] E2E tests
- [X] Local manual tests

### CODE REVIEW

File: apps/frontend/src/hooks/use-is-mobile.tsx [hunk 1]
```diff
@@ -1,9 +0,0 @@
-import { useWindowSize } from 'usehooks-ts';
-
-const MOBILE_SIZE = 640;
-const useIsMobile = (): boolean => {
```
Comment (Kroustille) [inline apps/frontend/src/hooks/use-is-mobile.tsx:4]: This hook is referenced in `copilot-instructions` and needs to be removed

**Review by Kroustille — APPROVED** [review]

**Comment by Kroustille** [comment]
Please double check but I think this keys can be safely removed:

```
PublicHomePage.Title
PublicHomePage.XtmPlatform.Cta
PublicHomePage.XtmPlatform.ImageAlt
PublicHomePage.XtmRoadmap.Next
PublicHomePage.XtmRoadmap.Now
PublicHomePage.XtmRoadmap.UnderConsideration
```
